### ISSUE TITLE
Doesn't work, either on the web or locally

### ISSUE THREAD

**Issue by brucehoult** [riscv-software-src/riscv-angel#17]
A number of sources say https://riscv.org/angel shoudl work, but it 404s.

The readme at https://github.com/riscv/riscv-angel says https://riscv.org/software-tools/riscv-angel/ and that page exists, but the link for "LAUNCH ANGEL" (http://52.32.189.224/angel-simulator/) doesn't work. The IP "52.32.189.224" belongs to AWS, but seems not to be reachable.

If I check out the project and try to launch locally then downloading the Linux image fails.

**Comment by sagark** [comment]
I'm not sure what's going on with the version hosted for the RISC-V website, hopefully whoever runs that sees the mailing list post. https://groups.google.com/a/groups.riscv.org/forum/#!topic/isa-dev/UzbUQWfI7Qg

I will add the instructions I posted there to the readme.

**Comment by sorear** [comment]
@sagark I think riscv.org is just taking the master branch readme; the site updated when I updated the master branch readme for qemu, so maybe you could write something appropriate for the riscv-angel master branch?

**Comment by sagark** [comment]
Good call, I've applied it there too. Although I think the website is doing something different to the ANGEL page, at least the "check out the demo ... " link doesn't match any of the readmes.

**Comment by sorear** [comment]
Actually I think @rickoco owns this, I'll email him tomorrow.

Might it be a better idea to point people at riscvemu?  It's flashier and more recently maintained (although the version on bellard.org is still using the old calling convention)

**Comment by sagark** [comment]
Yeah, that's fine with me

**Comment by davidak** [comment]
The site works for me. Can you recheck?

**Comment by miiichael** [comment]
Nope, https://riscv.org/software-tools/riscv-angel/ continues to be 404.
